### PR TITLE
Add Mako related files to build_tools/mako/config/

### PR DESCRIPTION
**Pull request by hanhanW**

### CODE REVIEW

File: mako_config/README.md
Comment (GMNGeoffrey) [inline mako_config/README.md:2]: Hmm having a personal fork for doing this is a bit gross...

File: mako_config/mobile-bert.config
Comment (GMNGeoffrey) [inline mako_config/mobile-bert.config:18]: go links are not public. This is the best public reference I can find https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.text_format

(Really??!)

File: mako_config/mobile-bert.config
Comment (GMNGeoffrey) [inline mako_config/mobile-bert.config:27]: What is update_benchmark.config? We don't have it checked in. Also this file *does* define benchmark_key. I assume you're copying from https://github.com/google/mako/blob/master/examples/benchmark_info/update_benchmark.config and https://github.com/google/mako/blob/master/examples/benchmark_info/create_benchmark.config

Please review all the copied documentation to ensure it is still relevant

File: mako_config/mobile-bert.config
Comment (GMNGeoffrey) [inline mako_config/mobile-bert.config:29]: Again not public

```suggestion
# For assistance using the command line tool see
https://github.com/google/mako/blob/master/docs/CLI.md.
```

File: mako_config/mobile-bert.config
Comment (GMNGeoffrey) [inline mako_config/mobile-bert.config:48]: What is the significance of the letters selected here?

**Review by GMNGeoffrey — CHANGES_REQUESTED** [review]
Let's put this under build_tools. Also ugh at how not-really-open-source this is...

File: mako_config/README.md
Comment (hanhanW) [reply mako_config/README.md:2]: yeah, I was testing it with my personal repo. Any idea where to put it?

File: mako_config/mobile-bert.config
Comment (hanhanW) [reply mako_config/mobile-bert.config]: I removed the comment. I think we don't need these in a config file. They can be found in Mako's example.

File: mako_config/mobile-bert.config
Comment (hanhanW) [reply mako_config/mobile-bert.config]: removed

File: mako_config/mobile-bert.config
Comment (hanhanW) [reply mako_config/mobile-bert.config:33]: They suggest to use one or two chars: https://github.com/google/mako/blob/master/spec/proto/mako.proto#L91

`c` stands for `cpu`, `g` stands for `gpu`, and `v` stands for `vmla`.

Comment (benvanik) [reply mako_config/mobile-bert.config:33]: hmm that doesn't scale very well :P what about vulkan and metal in this?
I'd prefer the overhead of ~3 characters (relative to all the rest of this proto junk it's insignificant) and something we have a chance in hell of ever decoding and keeping consistent. 'vlk' 'mtl' (both common abbreviations used in the ecosystem) 'vmla' 'cpu' etc seem better.

Comment (hanhanW) [reply mako_config/mobile-bert.config]: SG and done.

(I still have to figure out how to update runs with new keys before land and update the config, so I will merge this later if others look good.)

File: mako_config/README.md
Comment (GMNGeoffrey) [reply mako_config/README.md:2]: I guess we should probably create google/iree-mako if we really want a separate repo. IREE should not depend on your personal repo in any long-term capacity.

File: mako_config/project.config
Comment (GMNGeoffrey) [inline mako_config/project.config:24]: "owner_list contains a list of owners" is a useless comment, most of this references google-internal stuff, and we definitely should not enable global permissions.

```suggestion
```

But it would be useful to add a comment about what these owners are allowed to do. I assume they can create new benchmarks and owners of an individual benchmark can upload new runs to it?

File: mako_config/mobile-bert.config
Comment (GMNGeoffrey) [inline mako_config/mobile-bert.config:28]: I think we probably don't want a timestamp here? We should be tagging this by commit

Comment (hanhanW) [reply mako_config/mobile-bert.config:28]: This is for a run chart, not benchmark chart. E.g., https://mako.dev/run?run_key=5693387618385920&~c=1&~v=1&~g=1

For now, we only have one number which indicates the total execution time. I have been thinking more about this, we probably can make it be a profiling plot, so when click into a run, we will see a plot. Outside a run, we can aggregate the numbers (or just use the last one) and show in benchmark chart (https://mako.dev/benchmark?benchmark_key=5538704950034432)

In both cases, it is indeed TIMESTAMP.

The commit is attached as a "Hover Text" to a run, which is showed up in benchmark chart (when you move the curser on the data point).

File: mako_config/project.config
Comment (hanhanW) [reply mako_config/project.config]: I think it's better to remove it. A project owner actually have all the permissions on project and benchmarks under the project, e.g, create/update/delete benchmarks and runs. So I think it's okay to not add a comment.

File: mako_config/README.md
Comment (hanhanW) [reply mako_config/README.md:2]: SG, I think we can leave it as this way for now. I will figure out how to create google/iree-mako (or ask someone to help) and we can replace it.

File: mako_config/mobile-bert.config
Comment (hanhanW) [reply mako_config/mobile-bert.config]: I successfully updated the config, existing runs and current scripts used in CI.

File: mako_config/mobile-bert.config
Comment (hanhanW) [reply mako_config/mobile-bert.config]: Got you, this is great and I totally agree. Let's use `NUMERIC` now, and we can iterate on it later when we want to do more stuff. I actually plan to have more repetitions in one run, so we can make numbers more stable. `NUMERIC` makes sense in any cases.

**Review by GMNGeoffrey — APPROVED** [review]
Sorry missed this earlier, though I think I commented. Can we put this all under build_tools/mako/config?

**Comment by hanhanW** [comment]
> Sorry missed this earlier, though I think I commented. Can we put this all under build_tools/mako/config?

Sorry I missed the comment...

**Pull request merged**
